Xamarin.Forms.Platform.Android/Resources/values/styles.xml

```diff
@@ -2,7 +2,10 @@
 <resources>
   <style name="XamarinFormsMaterialTheme" parent="Theme.MaterialComponents.Light.DarkActionBar">
   </style>
-  <style name="XamarinFormsMaterialProgressBarHorizontal" parent="Widget.AppCompat.ProgressBar.Horizontal">
+  <style name="XamarinFormsMaterialProgressBarHorizontal" parent="android:Widget.ProgressBar.Horizontal">
+    <item name="android:indeterminateOnly">false</item>
+    <item name="android:minHeight">4dp</item>
+    <item name="android:progressDrawable">@drawable/MaterialProgressBar</item>
```

Comment:
Why did you opt to use the base control and a custom drawing?

Reply:
Just got reminded by Shane, Google has a _HARD_ set for 4dp. Of course, this is absolutely not true for iOS, but hey, why follow your own guides.